pandas/core/tools/numeric.py

```diff
@@ -35,17 +35,23 @@ def to_numeric(arg, errors="raise", downcast=None):
     Parameters
     ----------
     arg : scalar, list, tuple, 1-d array, or Series
+        Arg is consists of any kinds of type :
+        - scalar is any kind of numeric type data.
+        - list is any kind of '[]' dtypes.
+        - tuple is any kind of '()' dtypes.
+        - 1-d array is list dtypes that only have 1 list.
+        - Series is kind of pandas 1-d ndarray with axis labels.
     errors : {'ignore', 'raise', 'coerce'}, default 'raise'
         - If 'raise', then invalid parsing will raise an exception.
         - If 'coerce', then invalid parsing will be set as NaN.
         - If 'ignore', then invalid parsing will return the input.
-    downcast : {'integer', 'signed', 'unsigned', 'float'}, default None
+    downcast : {'int', 'signed', 'unsigned', 'float'}, default None
```

Comment:
Please change this back to ```integer```, you are completely right when it comes to the type of rhe variable that we want the python type name and not the pre-historic name, but in this case it's the value, and code below will check if ```downcast``` is equals to ```integer``` and not for ```int```

Reply:
alright, ill be back soon

Reply:
revert back to integer done, please kindly check :")